OldTests/src/test/java/com/tle/webtests/test/workflow/rejection/NewVersionTest.java

```diff
@@ -15,81 +16,111 @@
 import com.tle.webtests.pageobject.wizard.WizardPageTab;
 import com.tle.webtests.test.AbstractCleanupTest;
 
+
 @TestInstitution("workflow")
 public class NewVersionTest extends AbstractCleanupTest
 {
+	private static final String BACKTICK_PASSWORD = "``````";
```

Comment:
Since these are all `public static final String`, the whitespaces between each line should be removed.